### PR TITLE
Rename resource to log view and add docs

### PR DESCRIPTION
**Pull request by ssingamneni-sfx**

### CODE REVIEW

File: website/docs/r/logs_list_chart.html.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+layout: "signalfx"
+page_title: "SignalFx: signalfx_logs_view_chart"
+sidebar_current: "docs-signalfx-resource-logs-view-chart"
+description: |-
+  Allows Terraform to create and manage Logs View charts
+---
+
+# Resource: signalfx_logs_view_chart
+
+You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first. A logs view chart displays log lines in a table form in a dashboard adn shows you in detail what is happening and why.
```
Comment (tcarter-splunk) [inline website/docs/r/logs_list_chart.html.markdown]: You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first. A log view displays log lines in a table form in a dashboard and shows you in detail what is happening and why.

- I changed "adn" to "and" 
- "logs view chart" should be "log view"

File: website/docs/r/logs_list_chart.html.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+layout: "signalfx"
+page_title: "SignalFx: signalfx_logs_view_chart"
+sidebar_current: "docs-signalfx-resource-logs-view-chart"
+description: |-
+  Allows Terraform to create and manage Logs View charts
```
Comment (tcarter-splunk) [inline website/docs/r/logs_list_chart.html.markdown]: Allows Terraform to create and manage log views

(Note that we should not capitalize the feature name. We only capitalize product names. Also it is "log view" not "logs view")

File: website/docs/r/logs_list_chart.html.markdown [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+* `name` - (Required) Name of the chart.
+* `program_text` - (Required) Signalflow program text for the chart. More info at https://developers.signalfx.com/docs/signalflow-overview.
+* `description` - (Optional) Description of the chart.
+* `time_range` - (Optional) From when to display data. SignalFx time syntax (e.g. `"-5m"`, `"-1h"`). Conflicts with `start_time` and `end_time`.
+* `start_time` - (Optional) Seconds since epoch. Used for visualization. Conflicts with `time_range`.
+* `end_time` - (Optional) Seconds since epoch. Used for visualization. Conflicts with `time_range`.
+* `columns` - (Optional) The column headers to show on the logs list table.
+* `sort_options` - (Optional) The sorting options configuration to specify if the table needs to be sorted in a particular field.
+* `default_connection` - (Optional) The connection that the chart uses to fetch data. This could be Splunk Enterprise, SPlunk Enterprise Cloud or Observability Cloud.
```
Comment (tcarter-splunk) [inline website/docs/r/logs_list_chart.html.markdown]: "SPlunk Enterprise Cloud" should be "Splunk Enterprise Cloud" (P should not be capitalized)

**Review by tcarter-splunk — CHANGES_REQUESTED** [review]

File: website/docs/r/logs_list_chart.html.markdown (hunk 1 above)
Comment (ssingamneni-sfx) [reply website/docs/r/logs_list_chart.html.markdown]: done

File: website/docs/r/logs_list_chart.html.markdown (hunk 2 above)
Comment (ssingamneni-sfx) [reply website/docs/r/logs_list_chart.html.markdown]: done

File: website/docs/r/logs_list_chart.html.markdown (hunk 3 above)
Comment (ssingamneni-sfx) [reply website/docs/r/logs_list_chart.html.markdown]: done

File: website/docs/r/log_view.html.markdown [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+# Resource: signalfx_log_view
+
+You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first. A log view displays log lines in a table form in a dashboard and shows you in detail what is happening and why.
+
+## Example Usage
+
+```tf
+resource "signalfx_log_view" "mychart0" {
```
Comment (bgola-signalfx) [inline website/docs/r/log_view.html.markdown]: nit: `mychart0` --> `my_log_view0`?

File: website/docs/r/log_view.html.markdown [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+}
+```
+
+## Argument Reference
+
+The following arguments are supported in the resource block:
+
+* `name` - (Required) Name of the chart.
```
Comment (bgola-signalfx) [inline website/docs/r/log_view.html.markdown]: Change all occurrences of `chart` to `log view`?

**Review by bgola-signalfx — APPROVED** [review]

File: website/docs/r/log_view.html.markdown (hunk 5 above)
Comment (ssingamneni-sfx) [reply website/docs/r/log_view.html.markdown]: done

File: website/docs/r/log_view.html.markdown (hunk 4 above)
Comment (ssingamneni-sfx) [reply website/docs/r/log_view.html.markdown]: done

**Pull request merged**